nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -1618,11 +1618,12 @@ public void OptionalEmptinessHandlerWithSingleCustomPathTest() {
             "import javax.annotation.Nullable;",
             "import com.google.common.base.Function;",
             "public class TestNegative {",
+            "  class ABC { Optional<Object> ob = Optional.absent();} ",
             "  void foo() {",
-            "    Optional<Object> a = Optional.absent();",
+            "      ABC abc = new ABC();",
```

Comment:
Made these tests stronger, as wrong code was passing tests

Reply:
I am also not sure I get the change.